mergify_engine/actions/copy.py

```diff
@@ -0,0 +1,139 @@
+# -*- encoding: utf-8 -*-
+#
+#  Copyright © 2018 Mehdi Abaakouk <sileht@sileht.net>
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import re
+from urllib import parse
+
+import daiquiri
+
+import github
+
+import voluptuous
+
+from mergify_engine import actions
+from mergify_engine import duplicate_pull
+
+LOG = daiquiri.getLogger(__name__)
+
+
+def Regex(value):
+    try:
+        re.compile(value)
+    except re.error as e:
+        raise voluptuous.Invalid(str(e))
+    return value
+
+
+class CopyAction(actions.Action):
+    KIND = "copy"
+    SUCCESS_MESSAGE = "Pull request copy have been created"
```

Comment:
"has been"

The message for backports is pural, I guess it should be here too:

`Pull requests copies have been created`